### PR TITLE
Polymer 3 2018 12 21 145153

### PR DESCRIPTION
**Pull request by awikkerink**

### CODE REVIEW

**Comment by awikkerink** [comment]
ping @MichaelKovalchuk

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,39 +7,33 @@ jobs:
...
     docker:
       - image: circleci/node:8.11-browsers
 
     steps:
       - checkout
-      - run: npm install artifactory-bower-publisher
-      - run: ./node_modules/.bin/artifactory-bower-publisher -u $ARTIFACTORY_USER -p $ARTIFACTORY_PASS -a https://d2lartifacts.artifactoryonline.com/d2lartifacts/simple/bower-local/polymer -n d2l-content-icons
+      - run: curl -ks -u$ARTIFACTORY_USER:$ARTIFACTORY_PASS "https://d2lartifacts.artifactoryonline.com/d2lartifacts/api/npm/npm-local/auth/d2l" -o .npmrc
+      - run: sudo npm install -g publish
+      - run: publish
```
Comment (MichaelKovalchuk) [inline .circleci/config.yml]: we only need to use publish on the very first time. it doesn't check if the version is already exists.
i'd suggest running the command manually to register the package and remove these lines from circleci.

Comment (awikkerink) [reply .circleci/config.yml]: The `publish` command is supposed to do that check for you. https://www.npmjs.com/package/publish

Comment (MichaelKovalchuk) [reply .circleci/config.yml]: but it requires that the package is published with the `npm publish` on the very first time. 
i see, we are already using it like this:
publish: 0.6.0 in the devDependencies
    `- run: ./node_modules/.bin/publish` in  circle ci

Comment (awikkerink) [reply .circleci/config.yml]: Correct

Comment (MichaelKovalchuk) [reply .circleci/config.yml]: ready to be merged?

Comment (awikkerink) [reply .circleci/config.yml]: I'm going to bump the version then release it first

Comment (awikkerink) [reply .circleci/config.yml]: Done. Ready to be merged

**Pull request merged**